docs/signature/README.md

```diff
@@ -0,0 +1,228 @@
+# Notary V2 Signature Specification
+
+This section defines the signature file, which is in JSON format with no whitespaces. Its JSON schema is available at [schema.json](schema.json).
+
+## Signature Goals
+
+- Offline signature creation
+- Persistance within an [OCI Artifact][oci-artifacts] enabled, [distribution-spec][distribution-spec] based registry
+- Artifact and signature copying within and across [OCI Artifact][oci-artifacts] enabled, [distribution-spec][distribution-spec] based registries
+- Support public registry acquisition of content - where the public registry may host certified content as well as public, non-certified content
+- Support private registries, where public content may be copied to, and new content originated within
+- Air-gapped environments, where the originating registry of content is not accessable
+- Multiple signatures per artifact, enabling the originating vendor signature, public registry certification and user/environment signatures
+- Maintain the original artifact digest and collection of associated tags, supporting dev/ops deployment definitions
+
+## Signature
+
+A Notary v2 signature is clear-signed signature of manifest metadata, including but not limited to
+
+- [OCI Image Index](https://github.com/opencontainers/image-spec/blob/master/image-index.md)
+- [OCI Image Manifest](https://github.com/opencontainers/image-spec/blob/master/manifest.md)
+- [Docker Image Manifest List](https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list)
+- [Docker Image Manifest](https://docs.docker.com/registry/spec/manifest-v2-2/#image-manifest)
+
+### Signing an Artifact Manifest
+
+For any [OCI Artifact][oci-artifacts] submitted to a registry, an [OCI Manifest][oci-manifest] and an optional [OCI Manifest List/Index][oci-manifest-list] is required.
+
+The nv2 prototype signs a combination of:
+
+- Key properties
+- The target artifact manifest digest
+- *optional:* list of associated tags
+
+#### Generating a self-signed x509 cert
```

Comment:
Would we need to set up a CA infrastructure for x509 certificates so that they don’t have to be self-signed? If so, how will we manage root CAs and key revocation?

Reply:
We started with x509, but we don't expect this to be the final design. It was more something to get started to facilitate the discussions. There's an interesting conversation about what should be used to sign content.

Reply:
resolving this in lieu of the key management working group.